samples/tests/StaticHtmlComposites/CallWithChatCompositeJavascriptTest.test.ts

```diff
@@ -0,0 +1,39 @@
+import { test, expect } from '@playwright/test';
+import { encodeQueryData, createChatThreadAndUsers } from '../common/utils';
+import * as path from 'path';
+import dotenv from 'dotenv';
+
+dotenv.config({ path: path.join(__dirname, '..', '.env') });
+
+const SERVER_URL = 'http://localhost:8080/callWithChatComposite.html';
+const PARTICIPANTS = ['Jack'];
+
+test.describe('JS Bundle Test', () => {
+  test.beforeEach(async ({ page }) => {
+    // Go to the starting url before each test.
+    const user = await createChatThreadAndUsers(PARTICIPANTS);
+
+    // Add styles to remove the blinking cursor during snapshot tests
+    const styleContent = `
+    input {
+      color: transparent;
+      text-shadow: 0 0 0 black;
+    } 
+    input:focus {
+      outline: none;
+    }
+  `;
+
+    const qs = encodeQueryData(user[0]);
+    const url = `${SERVER_URL}?${qs}`;
+    await page.addStyleTag({ content: styleContent });
+    await page.goto(url);
```

Comment:
Btw, these logics are somehow overlapped (except the real test code, what about we move page.goto into each test() block and share the same file? - we can always separate them once there is any need)

So it would be like

```
const SERVER_BASE = 'http://localhost:8080/';

const calCompositePage = 'callComposite.html';

const callWithChatCompositePage = 'callWithChatComposite.html';
test.describe('JS Bundle Test', () => {
   test.beforeEach(async ({ page }) => {
   ...
   }
   test('Test 1', async ({ page }) => {
      page.goto(SERVER_BASE + callWithChatCompositePage);
...
   }
   test('Test 2', async ({ page }) => {
      page.goto(SERVER_BASE + callCompositePage);
...
   }
}
```